examples/polar_plots/polarplot/polarplot_10.cpp

```diff
@@ -0,0 +1,33 @@
+#include <algorithm>
+#include <cmath>
+#include <matplot/matplot.h>
+#include <vector>
+
+using namespace matplot;
+
+// https://en.wikipedia.org/wiki/Von_Mises_distribution
+double von_mises_distribution_pdf(double t_mean, double kappa, double t_dir) {
+    const double normalization = 0.5 / (pi * std::cyl_bessel_i(0.0, kappa));
+    const double cos_delta_t = std::cos(t_dir - t_mean);
+    return normalization * std::exp(kappa * cos_delta_t);
+}
+
+int main() {
+    const double mu = pi/4.;
+    const double kappa = 2.;
+    std::vector<double> thetas = linspace(0., 2. * pi, 200);
+    std::vector<double> rhos = transform(thetas, [&](auto theta) { return  von_mises_distribution_pdf(mu, kappa, theta); });
+    const auto max = *std::max_element(rhos.begin(), rhos.end());
+    const auto uniform_density = 1./(2.0 * pi);
+
+    auto fig = figure(true);
+    auto ax = fig->add_subplot(1, 1, 0);
+
+    ax->rticklabels({"uniform", "max (κ=2.)"});
+    ax->rticks({uniform_density, max});
+    ax->rtickangle(60.);
+    ax->polarplot(rhos);
+
```

Comment:
Without the fix, it is necessary to add `ax->r_axis().visible(true);` _after_ the `polarplot()` call in order for the custom tick labels to show up.

Reply:
Note: the resulting images are not identical: `ax->r_axis().visible(true);` has the additional (obvious) effect of making the r-axis visible which, at least in my case, might be undesirable.